.github/workflows/pr-platform.yml

```diff
@@ -1,14 +1,17 @@
 name: 'pr-platform'
 on:
```

Comment:
do we need to add an `on` event for the scheduled run? or does it fall under `workflow_dispatch`

Reply:
https://github.com/nasa/openmct/pull/6782/files/4cd56e48373f409dff148af770a4faf1e33b52ab#r1256537179